lib/src/clojure_lsp/kondo.clj

```diff
@@ -288,18 +288,14 @@
         (with-additional-config (settings/all db)))))
 
 (defn ^:private run-kondo! [config err-hint db]
-  (let [err-writer (java.io.StringWriter.)
-        out-writer (java.io.StringWriter.)]
+  (let [err-writer (java.io.StringWriter.)]
     (try
       (let [result (if (:api? db)
```

Comment:
Hmm... when run with `:api?` we weren't capturing stdout. But now we are, because `shared/capturing-stdout` wraps this call. So the kondo output, which was going to stdout, will now go to the logs. Is this a problem though? `kondo/run!` normally doesn't put anything on stdout.

Reply:
This is a fix!
For CLI at least this is good as we were logging unnecessary kondo logs even without --verbose flag, for API I think it's acceptable to lose that log too